pontoon/base/static/js/translate.js

```diff
@@ -1538,6 +1538,13 @@ var Pontoon = (function (my) {
           e.preventDefault();
           signIn();
         });
+
+        $('ul.links > li#sign-in').click(function (e) {
+          console.log('Reached here 1 !');
```

Comment:
I used these to debug, In the final rebased commit, i'll remove all of these :smile: